### ISSUE TITLE
Hangs, then fails when run from a different drive letter under Windows

### ISSUE THREAD

**Issue by jimbeveridge** [GoogleCloudPlatform/gsutil#258]
gsutil version: 4.10
SDK version: 0.9.51
Environment: Windows 7

I have my upload data stored on drive F:. I have the Cloud SDK (including gsutil) installed on drive C:.

If I run the following commands, gsutil hangs for ~30 seconds, then fails. The first line sets my default drive to F. Obviously this command could have been run from Drive C, but the lack of a meaningful error message made this problem challenging to solve.

```
F:
gsutil acl ch -u AllUsers:R gs://mybucket
```
Here's the stack dump:

```
ERROR 0315 19:56:58.915000 retry_decorator.py] Retrying in 1.04 seconds ...
Traceback (most recent call last):
  File "C:\Program Files\Google\Cloud SDK\google-cloud-sdk\platform\gsutil\third_party\retry-decorator\retry_decorator\retry_decorator.py", line 20, in f_retry
    return f(*args, **kwargs)
  File "C:\Program Files\Google\Cloud SDK\google-cloud-sdk\platform\gsutil\gslib\commands\acl.py", line 415, in ApplyAclChanges    fields=['acl', 'metageneration'])
  File "C:\Program Files\Google\Cloud SDK\google-cloud-sdk\platform\gsutil\gslib\cloud_api_delegator.py", line 170, in GetBucket
    return self._GetApi(provider).GetBucket(bucket_name, fields=fields)
  File "C:\Program Files\Google\Cloud SDK\google-cloud-sdk\platform\gsutil\gslib\gcs_json_api.py", line 311, in GetBucket
    self._TranslateExceptionAndRaise(e, bucket_name=bucket_name)
  File "C:\Program Files\Google\Cloud SDK\google-cloud-sdk\platform\gsutil\gslib\gcs_json_api.py", line 1154, in _TranslateExceptionAndRaise
    raise translated_exception
AccessDeniedException: AccessDeniedException: 401 Login Required
```

I tried to run gcloud auth login, which succeeds, but then repeating the above command gives:
```
You are attempting to access protected data with no configured
credentials. Please visit https://cloud.google.com/console#/project
and sign up for an account, and then run the "gcloud auth login"
command to configure gsutil to use these credentials.
```

**Comment by thobrla** [comment]
Would like to fix this, waiting on acceptance of https://github.com/boto/boto/pull/2823

**Comment by nitinsurana** [comment]
+1

**Comment by thobrla** [comment]
Fixed as of gsutil 4.13
